docs/release-notes/8.1.asciidoc

```diff
@@ -0,0 +1,40 @@
+[[release-notes-header-8.1.0]]
+== 8.1
+
+[discrete]
+[[release-notes-8.1.0]]
+=== 8.1.0
+
+[discrete]
+[[breaking-changes-8.1.0]]
+==== Breaking Changes
+// tag::breaking-changes[]
+// NOTE: The breaking-changes tagged regions are re-used in the Elastic Installation and Upgrade Guide. The pull tag is defined within this snippet so it properly resolves in the output.
+:pull: https://github.com/elastic/kibana/pull/
+There are no breaking changes in 8.1.0.
+// end::breaking-changes[]
+
+[discrete]
+[[features-8.1.0]]
+==== Features
+* Improves copy for the privilege check on the Endpoint page ({pull}124118[#124118]).
+* Introduces a new *Host risk classification* column in the All hosts table on the *Hosts* page. In addition, a new *Host by risk* tab has been added to the *Hosts* page and host detail pages. From the *Host by risk* tab, you can access an explanation of how a host’s risk is calculated and scored ({pull}122980[#122980], {pull}122586[#122586], {pull}122018[#122018], {pull}121075[#121075], {pull}120487[#120487], {pull}119734[#119734]).
+* Introduces the ability to  bulk edit rule index patterns and tags ({pull}122635[#122635]).
+* Expands Endpoint per-policy artifact assignment to include endpoint event filters and host isolation IP exceptions ({pull}121879[#121879], {pull}121632[#121632]).
+* Adds Rule Execution UUID ({pull}113058[#113058]).
+* Introduces case metrics that summarize alert information and response times ({pull}121336[#121336]).
+
+[discrete]
+[[bug-fixes-8.1.0]]
+==== Bug fixes and enhancements
+* Improves the performance of indicator match rules ({pull}123882[#123882], {pull}123677[#123677]).
+* Changes the default indicator index query of custom and prebuilt indicator match rules to `@timestamp >= "now-30d/d"` ({pull}123590[#123590]).
+* Improves the exceptions interface by replacing the exceptions modal with a flyout ({pull}123408[#123408]).
+* Alert details flyout enhancements:
+** Shows different “highlighted fields” in an alert’s details flyout based on its type, category, and code ({pull}123239[#123239]).
+** Adds overview cards with key data to the alert details flyout ({pull}120347[#120347]).
+** Allows you to aggregate alert data based on any aggregatable ECS field instead of just 10 preset options ({pull}120610[#120610]).
+* Speeds up the rules table by implementing an in-memory rules cache ({pull}119611[#119611]).
```

Comment:
Currently this is an optional technical preview feature -- should we mention this?

This feature also allows the user to sort the rules table on more columns, which is more of an outright new feature so we should probably add this to the Features section above.

Reply:
@pedro we could use your input here! It seems that #119611 added quite a bit in terms of enhancements to the **Rules** tab and **Rule Monitoring** tab on the Rules page. Does the current summary fully represent that? If not, what changes do you recommend? Also, do you have a preference for where this PR resides in the release notes? The actual release note label on the PR categorizes it as an enhancement, which is why we placed it here. However, if you'd prefer that these changes are more visible, and consider the changes significant enough that they warrant the "new feature" label, we can move it to the New Features section.

Reply:
Hi @nastasha-solomon, I would definitely favor moving this to the `Features` section because it's a highly requested enhancement.

As far as summarizing it, I would only highlight that the experimental feature allows users to to sort on all rule management columns. We don't need to highlight the performance enhancements described in the PR because there are trade offs with performance.